arrocke/gloss-translation
# fix: language permissions bug

## What has changed
- The app instantly recognizes self-made updates in the current user's language roles

## Connected Issues

closes #333

## QA Steps

1. As admin, add a new language
2. Confirm that you can't translate the language
3. Add yourself as a translator - confirm that you can translate in the language
4. Try removing and then adding "Translator" from/to your roles in the language - confirm that these actions appropriately update your translation permissions.
5. Now remove yourself from the language members - confirm that you **cannot** translate in the language

## Post-Deployment
